app/controllers/requests_controller.rb

```diff
@@ -40,6 +40,24 @@ def start
     redirect_to new_distribution_path(request_id: request.id)
   end
 
+  def print_unfulfilled
+    requests = current_organization
+      .requests
+      .includes(:item_requests, partner: [:profile])
```

Comment:
This should be `.includes(item_requests: :item)` to eager-load items as well.

Reply:
I looked at where I used `item_request.item` and saw it was only for the `item.name`. Since there is already `item_request.name`, I updated to use that, which seems it might even be preferable to `item.name` and removed the need to load `item` at all.